modules/ROOT/pages/to-delete-asset.adoc

```diff
@@ -1,27 +1,32 @@
-= Delete an Asset
+= Deleting an Asset
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+Keep your asset catalog maintained by deleting outdated or redundant assets. This makes is easier for consumers to find and reuse the assets they need. 
 
-You can delete an asset in the first seven days of its creation and reuse that asset's name, group, asset ID, and version. A delete within the first seven days is known as a _hard delete_.
+When deleting assets, admins determine if the Group ID, Asset ID, and Version (GAV) information for a deleted asset can be reused for a different asset. 
```

Comment:
Here it says `Group ID, Asset ID, and Version (GAV)`, but in  modules/ROOT/pages/managing-asset-deletion.adoc it says `GAV (group ID, asset ID, and version)`, for consistency can we use the same format in both files?